src/libraries/restore/runtime/runtime.depproj

```diff
@@ -113,8 +93,7 @@
           AfterTargets="AfterResolveReferences;FilterNugetPackages"
           Condition="'$(RuntimeFlavor)' == 'Mono'">
     <ItemGroup>
-      <StoreTestHostFiles Include="@(ReferenceCopyLocalPaths)" />
-      <ReferenceCopyLocalPaths Remove="@(ReferenceCopyLocalPaths)" />
+      <!-- nothing to do here -->
```

Comment:
instead you can just close the `Target` element:

```
<Target .... />
```

Reply:
Done.